src/logic/context/ContextManager.ts

```diff
@@ -0,0 +1,65 @@
+import {Context} from "../../data/Context";
+import {HotKeyAction} from "../../data/HotKeyAction";
+import {store} from "../../index";
+import {updateActiveContext} from "../../store/general/actionCreators";
+import * as _ from "lodash";
+
+export class ContextManager {
+    private static activeCombo: string[] = [];
+    private static actions: HotKeyAction[] = [];
+
+    public static init(): void {
+        window.addEventListener("keydown", ContextManager.onDown);
+        window.addEventListener("keyup", ContextManager.onUp);
+    }
+
+    public static register(context: Context, actions: HotKeyAction[]): void {
```

Comment:
register to chyba zła nazwa, bo ta funkcja ustawia context (mam rację?). więc raczej nazwiij to "switch" (jeśli się nie mylę)